launch/mavros_sitl.launch

```diff
@@ -0,0 +1,21 @@
+<launch>
+	<!-- vim: set ft=xml noet : -->
+	<!-- example launch script for PX4 based FCU's -->
+
+	<group ns="px4_multicopter">
```

Comment:
Do we want this in the px4_namespace? If I'm doing something offboard, wouldn't I talk to a mavros node located at root level?